src/Symfony/Component/AssetMapper/ImportMap/Resolver/JsDelivrEsmResolver.php

```diff
@@ -125,19 +134,34 @@ public function resolvePackages(array $packagesToRequire): array
      * records the package as a dependency, so it can be downloaded and
      * added to the importmap.
      */
-    private function parseJsDelivrImports(string $content, array &$dependencies, bool $download, bool $preload): string
+    private function parseJsDelivrImports(string $content, array &$dependencies, bool $download): string
     {
         // imports from jsdelivr follow a predictable format
-        $content = preg_replace_callback(self::IMPORT_REGEX, function ($matches) use (&$dependencies, $download, $preload) {
+        $content = preg_replace_callback(self::IMPORT_REGEX, function ($matches) use (&$dependencies, $download) {
             $packageName = $matches[1];
             $version = $matches[2];
 
-            $dependencies[] = new PackageRequireOptions($packageName, $version, $download, $preload);
+            $dependencies[] = new PackageRequireOptions($packageName, $version, $download);
 
             return sprintf('from"%s"', $packageName);
         }, $content);
 
         // source maps are not also downloaded - so remove the sourceMappingURL
-        return preg_replace('{//# sourceMappingURL=.*$}m', '', $content);
+        $content = preg_replace('{//# sourceMappingURL=.*$}m', '', $content);
+
+        return preg_replace('{/\*# sourceMappingURL=.* \*/}m', '', $content);
+    }
+
+    private function addCssEntrypoint(string $packageName, string $version, array &$packagesToRequire, bool $download): void
+    {
+        $response = $this->httpClient->request('GET', sprintf(self::URL_PATTERN_ENTRYPOINT, $packageName, $version));
+        $entrypoints = $response->toArray()['entrypoints'] ?? [];
```

Comment:
could there be a way to queue these requests instead of resolving them synchronously? the other requests are done concurrently, it'd be nice to keep this as it provides the best DX